QUICKSTART.md

```diff
@@ -0,0 +1,214 @@
+});
+
+rep.subscribe(
+  async tx => await tx.scanAll(),
+  {
+    onData: result => {
+      const tbody = document.querySelector('tbody');
+      tbody.innerHTML = '';
+      for (const ([k, v]) of result) {
+        const row = document.createElement('tr');
+        const td1 = document.createElement('td');
+        const td2 = document.createElement('td');
+        td1.textContent = k;
+        td2.textContent = JSON.stringify(v);
+        row.appendChild(td1);
+        row.appendChild(td2);
+        tbody.appendChild(row);
+      }
+    },
+  }
+);
+```
+
+Replicache polls your Client View on a timer and updates the UI automatically.
+
+Notes:
+* If you use webpack 4, you need to configure it to be able to successfully load Replicache. See [Webpack 4 Setup](TODO) for details.
+* In production, you can use our hosted Diff Server instead of a local one. See [Create Hosted Diff Server Account](TODO) for details.
+* You can adjust the frequency of the polling using the [syncInterval](TODO) property. However, polling is typically only used for development. In production, we recommend using a [web socket poke](TODO) to tell the client when to sync.
+
+## 🏎 Batch Endpoint
+
+Add a `/replicache-batch` endpoint to your API server that accepts and processes batches of mutations that look like this:
+
+```
+{
+  "clientID": "CB94867E-94B7-48F3-A3C1-287871E1F7FD",
+  "mutations": [
+    {
+      "id": 7,
+      "name": "createItem",
+      "args": {
+        "key": "create-item-0.4420193",
+        "value": "foo",
+      }
+    },
+    {
+      "id": 8,
+      "name": "createItem",
+      "args": {
+        "key": "create-item-0.21301",
+        "value": "bar",
+      }
+    },
+  ]
+}
+```
+
+See [Replicache Upstream Sync](https://github.com/rocicorp/replicache/blob/main/SERVER_SETUP.md#step-4-upstream-sync) for instructions on how to implement this endpoint.
+
+### 🛫 Client-Side Mutations
+
+Add the following code to the bottom of the script block from Step 4.
+
+```js
+const createItem = rep.register('createItem', async (tx, {key, value}) => {
+  await tx.put(key, value);
+});
+
+window.onload = () => {
+  var button = document.createElement('button');
+  button.onclick = () => {
+    createItem(`new-item-${Math.random()}`, 'foo');
+  };
+};
+```
+
+When you tap the button the change happens immediately on the client, but is
+momentarily later sent to the server. You can verify it by turning the network
+off in web inspector or slowing it down.
+
+## 🚀 Next Steps
+
+That's it! You've built a fully-functioning Replicache-powered app against your own backend. What will you do next?
+
+- [Check out the full version of this sample](https://github.com/rocicorp/replicache-sdk-js/tree/main/sample/lit-todo)
+- [Learn how to add Replicache support to your own backend service](https://github.com/rocicorp/replicache/blob/main/SERVER_SETUP.md)
+- [Check out the richer React/Babel/GCal sample](https://github.com/rocicorp/replicache-sdk-js/tree/main/sample/cal)
```

Comment:
This sample does not work for me. Maybe link to sample/redo for a React example?

Reply:
ok!